c10/util/Flags.h

```diff
@@ -208,7 +208,7 @@ C10_DECLARE_REGISTRY(C10FlagsRegistry, C10FlagParser, const std::string&);
   C10_DEFINE_typed_var(std::string, name, default_value, help_str)
 
 // DECLARE_typed_var should be used in header files and in the global namespace.
-#define C10_DECLARE_typed_var(type, name) C10_IMPORT extern type FLAGS_##name
+#define C10_DECLARE_typed_var(type, name) C10_API extern type FLAGS_##name
```

Comment:
Again, why?

Reply:
If C10_DECLARE_typed_var is used in libc10 headers, we prefer dllexport rather than dllimport. This change eliminates some dll inconsistent warnings.